src/content/docs/blog/biome-v2-0-beta.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Biome v2.0 beta
+description: A major milestone in the making
+summary: |
+  Biome 2.0 will be packed with major features and many smaller fixes, rules, and other improvements.
+  There's a lot to unpack, and we request the community's help testing this beta, so the final
+  release can be as smooth as possible.
+authors:
+  - arendjr
+  - team
+date: 2025-03-31
+cover:
+  light: "@/assets/blog/roadmap-2024/banner-light.png"
+  dark: "@/assets/blog/roadmap-2024/banner-dark.png"
+  alt: The brand of the project. It says "Biome, toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+---
+
+After hard work from our team, Biome's long-awaited 2.0 release is nearing completion. It will be packed with many large features, so we would like your help testing it with a public beta!
+
+If you would like to try it out, you can update Biome and migrate your configuration using the following commands:
+
+```shell
+npm install --save-dev --save-exact @biomejs/biome@beta
+npx @biomejs/biome migrate
+```
+
+## New features
+
+While the final 2.0 release may still have small changes in its final feature set, here's what you can expect in the beta:
+
+- **Plugins:** You can write custom lint rules using GritQL.
+- **Domains:** Domains help to group lint rules by technology, framework, or well, domain. Thanks to domains, your default set of recommended lint rules will only include those that are relevant to your project.
```

Comment:
```suggestion
- **Linter domains:** Domains help to group lint rules by technology, framework, or well, domain. Thanks to domains, your default set of recommended lint rules will only include those that are relevant to your project.
```

Just a suggestion to frame the name "domain" in the right context

Reply:
I think I'll leave this as is. The context is clear right after, and "Domains" is much stronger terminology than "Linter domains", especially since there's no real room for confusion.